cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -435,6 +437,47 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
             return updateStatusPromise.future();
         }
 
+        /**
+         * Sets the initial status when the Kafka resource is created and the cluster starts deploying.
+         *
+         * @return
+         */
+        Future<ReconciliationState> initialStatus() {
+            Promise<ReconciliationState> initialStatusPromise = Promise.promise();
+
+            crdOperator.getAsync(namespace, name).setHandler(getRes -> {
+                if (getRes.succeeded())    {
+                    Kafka kafka = getRes.result();
+
+                    if (kafka != null && kafka.getStatus() == null) {
+                        log.debug("{}: Setting the initial status for a new resource", reconciliation);
+
+                        Condition deployingCondition = new ConditionBuilder()
+                                .withLastTransitionTime(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").format(dateSupplier()))
+                                .withType("NotReady")
+                                .withStatus("True")
+                                .withReason("Deploying")
```

Comment:
We all know what "deploying" means, but it's not really a term defined in the docs. Would `InitialReconciliation` be better?

Reply:
To be honest, because it'a about the initial status of a Kafka cluster, I would understand "Deploying" much better than "InitialReconciliation" (which sounds to me more related to how an operator works, so reconciling "things", compared to a Kafka cluster which is deployed).

Reply:
I can live with both. I would think that Deploying is more universal than InitialReconciliation and might be better to understand by regular users. But I can change it if we agree on it.

Reply:
I am for "Deploying" :-)

Reply:
I changed this to _Creating_ which should be clear to everyone.